java/ql/test/library-tests/dataflow/range-analysis-inline/B.java

```diff
@@ -0,0 +1,90 @@
+public class B {
+    public int forloop() {
+        int result = 0;
+        for (int i = 0; i < 10; i++) {// $ bound="i in [0..10]" bound="i in [0..9]"
```

Comment:
Add linebreaks to clarify the match between expressions and expectations.
```suggestion
        for (int i = 0;
            i < 10; // $ bound="i in [0..10]"
            i++) { // $ bound="i in [0..9]"
```

Reply:
Nice 👍